static/css/rue89.css

```diff
@@ -0,0 +1,134 @@
+a:link,
```

Comment:
@smarnach Overriding the bundle CSS files was how I was told this would be done from the beginning.
I am honestly not sure right now which solution would be the easier to maintain, you might have more perspective on the matter than I do.
I think in the end @antoviaque and @srcmax will be the ones taking the decision.